tiatoolbox/tools/patchextraction.py

```diff
@@ -36,6 +37,13 @@ class PatchExtractor(ABC):
         input_img(str, pathlib.Path, :class:`numpy.ndarray`): input image for
           patch extraction.
         patch_size(int or tuple(int)): patch size tuple (width, height).
+        input_mask(str, pathlib.Path, :class:`numpy.ndarray`, or :obj:WSIReader):
+          input mask that is used for position filtering when extracting patches
+          i.e., patches will only be extracted based on the highlighted regions in
+          the input_mask. input_mask can be either path to the mask, a numpy
+          array, :class:`VirtualWSIReader`, or 'auto`. In case of 'auto', a
```

Comment:
adjust according to change in option to add `otsu` and `morphological`

Reply:
resolved in 137e25c758490f665c967a81f0263c34f5d17677